etc/config.xml

```diff
@@ -10,7 +10,7 @@
         <fastsimpleimport>
             <default>
                 <ignore_duplicates>0</ignore_duplicates>
-                <behavior>append</behavior>
+                <behavior>add_update</behavior>
```

Comment:
This is a breaking change. To be honest, I am not sure if we should do this.

Reply:
Good catch. No, I wouldn't change the default behavior - we'd need to introduce a 2.0 then, and I don't think that would be worth it.
Everything else in this PR looks good for me.